docs/proposals/edgemesh-design.md

```diff
@@ -76,6 +76,149 @@ Add two endpoints definition:
 |ServiceBus|Target|Request Forward|
 |Router|Source|Resutst Forward, Service Discovery, Load Balance|
 
+<img src="../images/proposals/router-design.png">
+
+* R1: EdgeController watches on k8s apiserver for Services/Endpoints.
+* R2: EC get the service definition and create router rule definition and save in DB(sqlite).
```

Comment:
It implies Edge Controller is coupling with router rules. AFAIK, Edge controller is responsible for taking care of the status of node and application. Can we just let router watch K8S Service & Endpoints and create the rules?

Reply:
we can create a new controller act like kubeproxy then

Reply:
@m1093782566 , The idea is to keep controller separate from router. Named it ServiceController who will watch on Services/Endpoints and create rules , Thanks